pkg/network/tlsSecurityProfile.go

```diff
@@ -0,0 +1,17 @@
+package network
+
+import (
+	"strings"
+
+	ocpv1 "github.com/openshift/api/config/v1"
+)
+
+func SecurityProfileSpec(profile *ocpv1.TLSSecurityProfile) (string, ocpv1.TLSProtocolVersion) {
+	if profile == nil {
+		return "", ""
+	}
+	if profile.Custom != nil {
+		return strings.Join(profile.Custom.TLSProfileSpec.Ciphers, ","), profile.Custom.TLSProfileSpec.MinTLSVersion
+	}
+	return strings.Join(ocpv1.TLSProfiles[profile.Type].Ciphers, ","), ocpv1.TLSProfiles[profile.Type].MinTLSVersion
+}
```

Comment:
Make this functions just select cipher suites and TLS version not converting formats

```suggestion
func SelectCIpherSuitesAndMinTLSVersion(profile *ocpv1.TLSSecurityProfile) ([]string, ocpv1.TLSProtocolVersion) {
	if profile == nil {
		return "", ""
	}
	if profile.Custom != nil {
		return profile.Custom.TLSProfileSpec.Ciphers, profile.Custom.TLSProfileSpec.MinTLSVersion
	}
	return ocpv1.TLSProfiles[profile.Type].Ciphers, ocpv1.TLSProfiles[profile.Type].MinTLSVersion
}
```

Reply:
Also kubemacpool is going to understand the [controller-runtime knob](Kubemacpool is going to understand the controller-runtime knob, can we convert it here ? 

https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/webhook/server.go#L190-L206) can we convert it here ?

Reply:
Done